components/CollectionOfferModal.tsx

```diff
@@ -28,6 +28,8 @@ const SOURCE_ID = process.env.NEXT_PUBLIC_SOURCE_ID
 const FEE_BPS = process.env.NEXT_PUBLIC_FEE_BPS
 const FEE_RECIPIENT = process.env.NEXT_PUBLIC_FEE_RECIPIENT
 
+const ARTBLOCKS = '0xa7d8d9ef8d8ce8992df33d8b8cf4aebabd5bd270'
```

Comment:
ArtBlocks is actually two contracts:
- 0xa7d8d9ef8d8ce8992df33d8b8cf4aebabd5bd270
- 0x059edd72cd353df5106d2b9cc5ab83a52287ac3a